### PR TITLE
ACS-3361 Support for inclusionType.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

**Comment by tpage-alfresco** [comment]
This requires https://github.com/Alfresco/alfresco-tas-restapi/pull/221 to be merged/released/included so that we don't have a SNAPSHOT dependency.

File: remote-api/src/main/java/org/alfresco/rest/api/impl/rules/RuleSetLoader.java [hunk 1]
```diff
@@ -45,15 +53,32 @@
...
+                if (parentRef.equals(folderNodeRef))
+                {
+                    ruleSet.setInclusionType(OWNED);
+                }
+                else
+                {
+                    boolean linked = nodeService.getParentAssocs(ruleSetNodeRef)
+                                           .stream().map(ChildAssociationRef::getParentRef)
+                                           .anyMatch(folderNodeRef::equals);
+                    ruleSet.setInclusionType(linked ? LINKED : INHERITED);
```
Comment (krdabrowski) [inline remote-api/src/main/java/org/alfresco/rest/api/impl/rules/RuleSetLoader.java:79]: I'm wondering if there exists a case when rule set could be linked and inherited in the same time. What should be the inclusion value for this case?

Comment (tpage-alfresco) [reply remote-api/src/main/java/org/alfresco/rest/api/impl/rules/RuleSetLoader.java:79]: The comment and extra unit tests are now added.

Comment (krdabrowski) [reply remote-api/src/main/java/org/alfresco/rest/api/impl/rules/RuleSetLoader.java:79]: Thanks for clarification and comment

**Review by krdabrowski — APPROVED** [review]

**Pull request merged**